tarantool/connection.py

```diff
@@ -1751,3 +1957,771 @@ def _check_features(self):
+    
+    def crud_len(self, space_name: str, opts: dict={}) -> int:
+        """
+        Gets the number of tuples in the space through 
+        the `crud <https://github.com/tarantool/crud#len>`__.
+
+        :param space_name: The name of the target space.
+        :type space_name: :obj:`str`
+
+        :param opts: The opts for the crud module.
+        :type opts: :obj:`dict`, optional
+
+        :rtype: :obj:`int`
+
+        :raise: :exc:`~tarantool.error.CrudModuleError`,
+            :exc:`~tarantool.error.DatabaseError`
+        """
+
+        assert isinstance(space_name, str)
+        assert isinstance(opts, dict)
+
+        crud_resp = call_crud(self, "crud.len", space_name, opts)
+
+        # In absence of an error, crud does not give variable err as nil (as in most cases).
+        if len(crud_resp) != 1:
+            raise CrudModuleError(None, CrudError(crud_resp[1]))
+
+        return crud_resp[0]
+
+    def crud_storage_info(self, opts: dict={}) -> dict:
+        """
+        Gets storages status through the 
+        `crud <https://github.com/tarantool/crud#storage-info>`__.
+
+        :param opts: The opts for the crud module.
+        :type opts: :obj:`dict`, optional
+
+        :rtype: :obj:`dict`
+
+        :raise: :exc:`~tarantool.error.CrudModuleError`,
+            :exc:`~tarantool.error.DatabaseError`
+        """
+
+        assert isinstance(opts, dict)
+
+        crud_resp = call_crud(self, "crud.storage_info", opts)
+
+        # In absence of an error, crud does not give variable err as nil (as in most cases).
+        if len(crud_resp) != 1:
+            raise CrudModuleError(None, CrudError(crud_resp[1]))
+
+        return crud_resp[0]
+
+    def crud_count(self, space_name: str, conditions: list=[], opts: dict={}) -> int:
+        """
+        Gets rows count through the 
+        `crud <https://github.com/tarantool/crud#count>`__.
+
+        :param space_name: The name of the target space.
+        :type space_name: :obj:`str`
+
+        :param conditions: The conditions for the crud module.
+        :type conditions: :obj:`list`, optional
+
+        :param opts: The opts for the crud module.
+        :type opts: :obj:`dict`, optional
+
+        :rtype: :obj:`int`
+
+        :raise: :exc:`~tarantool.error.CrudModuleError`,
+            :exc:`~tarantool.error.DatabaseError`
+        """
+
+        assert isinstance(space_name, str)
+        assert isinstance(conditions, (tuple, list))
+        assert isinstance(opts, dict)
+
+        crud_resp = call_crud(self, "crud.count", space_name, conditions, opts)
+
+        if crud_resp[1] is not None:
+            raise CrudModuleError(None, CrudError(crud_resp[1]))
+
+        return crud_resp[0]
+
+    def crud_stats(self, space_name=None) -> CrudResult:
+        """
+        Gets statistics through the 
+        `crud <https://github.com/tarantool/crud#statistics>`__.
+
+        :param space_name: The name of the target space.
+        :type space_name: :obj:`str`
```

Comment:
```suggestion
        :type space_name: :obj:`str`,Optional
```

You're also missing type hint.

Reply:
Done